openapi-cli/src/main/java/io/ballerina/openapi/generators/client/mime/MultipartFormData.java

```diff
@@ -0,0 +1,159 @@
+package io.ballerina.openapi.generators.client.mime;
```

Comment:
Missing licence header :)

Reply:
Added in https://github.com/ballerina-platform/openapi-tools/commit/bfcb193a81bf7f406a52321d11e64ad7920ca61f